### PR TITLE
[SELS-TASK] Implemented Edit Word and Choices Function

### PR DESCRIPTION
**Pull request by christanplaza**
Description: This PR is for the implementation of Edit Word and Choices Function for the Admin side. 

[Commands]
php artisan migrate:fresh --seed

[Pre-condition]
- Login as Admin (email: admin@gmail.com, password: securepassword)
- Go to /admin/categories
- Click on any category
- Edit any word
- Modify Fields (You can choose how many to modify, and the radiobutton stands for the correct answer between the choices)

[Expected Output]
- Should redirect you to the Category Page
- If you click on the word, the choices should be updated (if you updated any of them)
- The highlight for the answer should change (if you updated the correct answer)

[Notes]
The rest of the CRUD operations will soon follow.

### CODE REVIEW

**Comment by jeremiahC** [comment]
@christanplaza 
If edit only one choice from choices page, there is error validation
[Image from Gyazo](https://gyazo.com/26d89b60a2dd69d79e763c746563554b)

it should still update even though I will not edit other choices

**Comment by christanplaza** [comment]
> @christanplaza
> If edit only one choice from choices page, there is error validation
> [Image from Gyazo](https://gyazo.com/26d89b60a2dd69d79e763c746563554b)
> 
> it should still update even though I will not edit other choices

Will resolve this in a bit.

**Comment by christanplaza** [comment]
> @christanplaza
> If edit only one choice from choices page, there is error validation
> [Image from Gyazo](https://gyazo.com/26d89b60a2dd69d79e763c746563554b)
> 
> it should still update even though I will not edit other choices

Resolved in 5398299

File: resources/views/admin/word/edit.blade.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                                    <button type="submit" class="btn btn-success float-right">Edit Word</button>
+                                </div>
+                            </div>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </div>
+
+    <div class="col-md-12">
```
Comment (jeremiahC) [inline resources/views/admin/word/edit.blade.php:49]: fix indentation

**Review by jeremiahC — APPROVED** [review]

**Pull request merged**